### PR TITLE
*: add a changelog file to keep track of changes in each release.

### PR DESCRIPTION
**Pull request by rithujohn191**

### CODE REVIEW

**Comment by Quentin-M** [comment]
I feel like this release is much more than that.

e.g.
- CLUO
- self-hosted etcd and its UI
- TerraForm bare-metal + TerraForm AWS backend

**Comment by rithujohn191** [comment]
Expanded the list. Please let me know if I missed anything

File: changes.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+* TerraForm support for both bare-metal and AWS.
+* Support for self-hosted etcd which can be enabled via the UI.
+* Container Linux Update Operator(CLUO) is on by default.
+* Updates to Kubernetes v1.6.2.
+* Updates to Dex v2.4.1.
+* Updates to Console v1.5.2.
+* Updates to bootkube v0.4.2.
+* GUI Installer with terraform on AWS and bare-metal.
+* Control plane is more HA due to new leveraged 1.6 features.
+* API server-to-etcd communication is secured over TLS.
```
Comment (sym3tri) [inline changes.md]: What new features are in Dex? SAML?

Console changes too...

File: changes.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+* TerraForm support for both bare-metal and AWS.
+* Support for self-hosted etcd which can be enabled via the UI.
+* Container Linux Update Operator(CLUO) is on by default.
+* Updates to Kubernetes v1.6.2.
+* Updates to Dex v2.4.1.
+* Updates to Console v1.5.2.
+* Updates to bootkube v0.4.2.
+* GUI Installer with terraform on AWS and bare-metal.
+* Control plane is more HA due to new leveraged 1.6 features.
```
Comment (Quentin-M) [inline changes.md]: it's more about having full workload separation, let me rephrase after my release.

File: changes.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Tectonic Changelog
+
+## Tectonic 1.6.2-tectonic.1 (2017-04-10)
+
+* TerraForm support for both bare-metal and AWS.
```
Comment (Quentin-M) [inline changes.md]: Switch provisioning methods on AWS/Bare-Metal to TerraForm exclusively

File: changes.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+* Support for self-hosted etcd which can be enabled via the UI.
```
Comment (Quentin-M) [inline changes.md]: Introduce experimental support for self-hosted etcd using its operator, and associated UI.

File: changes.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+* Container Linux Update Operator(CLUO) is on by default.
```
Comment (Quentin-M) [inline changes.md]: Introduce the Container Linux Updator Operator (CLUO)

(we never had it before)

**Comment by Quentin-M** [comment]
Remove locksmithd, etcd-gateway

File: changes.md (hunk 2 above)
Comment (Quentin-M) [inline changes.md]: Segregate control-plane / user workloads to master / worker nodes respectively

**Comment by Quentin-M** [comment]
Add support for TLS communications when using an external etcd cluster

**Comment by Quentin-M** [comment]
Lock down various network endpoints? -- not sure of the wording here (kubelet, etcd...)

**Comment by Quentin-M** [comment]
Enable audit-logs for the API Server

**Comment by Quentin-M** [comment]
Fix an issue where new nodes started automatically by auto-scalers would start with an outdated version of kubelet

**Comment by Quentin-M** [comment]
Add the ability for developers to automate the deployment and management of Tectonic clusters using the CLI

**Comment by Quentin-M** [comment]
Remove final manual installation step

**Comment by amrutac** [comment]
@rithujohn191: Here are the release notes for console, you can skip the `fixes` section

**Comment by rithujohn191** [comment]
@sym3tri @Quentin-M addressed all feedback

File: changes.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+# Tectonic Changelog
+
+## Tectonic 1.6.2-tectonic.1 (2017-04-10)
+
+* Switches provisioning methods on AWS & Bare-Metal to Terraform exclusively.
+* Adds support for customizing the Tectonic infrastructure via terraform.
```
Comment (Quentin-M) [inline changes.md]: Terraform (sorry, let's respect their brand)

File: changes.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+* Introduces experimental support for self-hosted etcd using its operator, and associated UI.
+* Adds Container Linux Update Operator(CLUO).
+* Updates to Kubernetes v1.6.2.
+* Updates to bootkube v0.4.2.
+* GUI Installer with terraform on AWS and bare-metal.
```
Comment (Quentin-M) [inline changes.md]: same here

**Comment by rithujohn191** [comment]
Done :+1:
